service/iot-service-client/src/main/java/com/microsoft/azure/sdk/iot/service/jobs/JobClient.java

```diff
@@ -177,7 +177,6 @@ public JobClient(String hostName, AzureSasCredential azureSasCredential, JobClie
      * @throws IOException if the function cannot create a URL for the job
      * @throws IotHubException if the http request failed
      */
-    @SuppressWarnings("deprecation")
```

Comment:
What was the deprecation attribute getting us?

Reply:
This was a "redundant suppression"